group-coordinator/src/main/java/org/apache/kafka/coordinator/group/share/ShareCoordinator.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.kafka.coordinator.group.share;
 
 import org.apache.kafka.common.annotation.InterfaceStability;
+
```

Comment:
Generally the AK style is not to have blank lines in the middle of the o.a.k imports. Please remove it.